### PR TITLE
Ceus initialization

### PR DESCRIPTION
**Pull request by berkserbet**
### Why:
* Completed ceus pipeline including loadshape plotting.

### What:
* Needed files have been placed in S3

### Checklist
- [x] Finished My Changes
- [ ] Automated Unit Tests

### CODE REVIEW

**Comment by dchassin** [comment]
Normalized load shapes: 
- Abilene TX and Wichita Falls TX is missing invalid (all three are the same and have simultaneous heating and cooling.
- The vertical axis should be rescale to the winter max when it exceeds the summer max. It current rescales to 2.0.
- Display the units, which I think are "pu summer peak"

Total Loadshapes:
- Abilene and Wichita Falls are invalid
- Display the units, which I think we “W/home”.
- Most TX city baseloads are suspiciously low.

**Comment by dchassin** [comment]
The following output is received:
~~~
(venv_loadinsight) PC93693:loadinsight dchassin$ python init.py -d
Running LCTK with DEBUG set to True
2019-06-26 07:51:07,950 [MainThread  ] [INFO ]  Starting the LCTK main program
2019-06-26 07:51:09,569 [MainThread  ] [INFO ]  running task undiscount_gas_task
2019-06-26 07:51:09,569 [MainThread  ] [INFO ]  Reading ceus_cleandata.csv
2019-06-26 07:51:09,569 [MainThread  ] [INFO ]  Attempting to load ceus_cleandata.csv from S3
2019-06-26 07:51:10,069 [MainThread  ] [ERROR]  File ceus_cleandata.csv does not exist in S3 under lctk.data
Traceback (most recent call last):
  File "/Volumes/DATA/GitHub/loadinsight/generics/artifact.py", line 78, in _read_from_s3
    s3.download_file(base.REMOTE_PATH, f'rbsa/{filename}', temp_filename)
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/boto3/s3/inject.py", line 172, in download_file
    extra_args=ExtraArgs, callback=Callback)
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/boto3/s3/transfer.py", line 307, in download_file
    future.result()
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/s3transfer/futures.py", line 106, in result
    return self._coordinator.result()
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/s3transfer/futures.py", line 265, in result
    raise self._exception
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/s3transfer/tasks.py", line 255, in _main
    self._submit(transfer_future=transfer_future, **kwargs)
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/s3transfer/download.py", line 345, in _submit
    **transfer_future.meta.call_args.extra_args
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/botocore/client.py", line 661, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (404) when calling the HeadObject operation: Not Found
ERROR:root:An error occurred (404) when calling the HeadObject operation: Not Found
Traceback (most recent call last):
  File "init.py", line 122, in <module>
    execute_lctk(sys.argv[1:])
  File "init.py", line 106, in execute_lctk
    ceus_pipeline.execute()
  File "/Volumes/DATA/GitHub/loadinsight/pipelines/ceus/ceus.py", line 202, in execute
    self.pipeline.run()
  File "/Volumes/DATA/GitHub/loadinsight/generics/pipeline.py", line 45, in run
    pipeline_task.run()
  File "/Volumes/DATA/GitHub/loadinsight/generics/task.py", line 44, in run
    self.run_result = self.task_function()
  File "/Volumes/DATA/GitHub/loadinsight/pipelines/ceus/tasks/undiscount_gas.py", line 33, in _task
    data_map = self._get_data()
  File "/Volumes/DATA/GitHub/loadinsight/pipelines/ceus/tasks/undiscount_gas.py", line 30, in _get_data
    return self.load_data(self.my_data_files)
  File "/Volumes/DATA/GitHub/loadinsight/generics/artifact.py", line 113, in load_data
    data_dict[filename] = self._read_file(filename)
  File "/Volumes/DATA/GitHub/loadinsight/generics/artifact.py", line 55, in _read_file
    self._read_from_s3(filename)
  File "/Volumes/DATA/GitHub/loadinsight/generics/artifact.py", line 85, in _read_from_s3
    raise e
  File "/Volumes/DATA/GitHub/loadinsight/generics/artifact.py", line 78, in _read_from_s3
    s3.download_file(base.REMOTE_PATH, f'rbsa/{filename}', temp_filename)
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/boto3/s3/inject.py", line 172, in download_file
    extra_args=ExtraArgs, callback=Callback)
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/boto3/s3/transfer.py", line 307, in download_file
    future.result()
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/s3transfer/futures.py", line 106, in result
    return self._coordinator.result()
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/s3transfer/futures.py", line 265, in result
    raise self._exception
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/s3transfer/tasks.py", line 255, in _main
    self._submit(transfer_future=transfer_future, **kwargs)
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/s3transfer/download.py", line 345, in _submit
    **transfer_future.meta.call_args.extra_args
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Volumes/DATA/Anaconda/anaconda3/envs/venv_loadinsight/lib/python3.6/site-packages/botocore/client.py", line 661, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (404) when calling the HeadObject operation: Not Found
An error occurred (404) when calling the HeadObject operation: Not Found
~~~

File: pipelines/rbsa/rbsa.py [hunk 1]
```diff
@@ -80,68 +80,117 @@ def generate_result_plots(self):
...
                 image_index += 1
+
+        logger.info('GENERATING LOADSHAPE PLOTS')
+
+        image_index = 0
+        for idx, city in enumerate(loadshapes.zipcode.unique()):
+            city_df = loadshapes.loc[loadshapes.zipcode == city]
+            max_total = city_df[['Heating', 'Cooling'] + base_enduses].sum(axis=1).max()
+            if max_total <= 1: max_val = 1
+            else: max_val = int(max_total) + 1
```
Comment (jongoncalves) [inline pipelines/rbsa/rbsa.py]: There's a lot of areas where this code can be simplified. I put the suggestion somewhere else, however, other than that, this PR looks good.

Comment (berkserbet) [reply pipelines/rbsa/rbsa.py]: Fixed all of it and added ceus plot labels

**Pull request merged**